package.json

```diff
@@ -2,12 +2,10 @@
   "name": "webhook-provisioner",
   "version": "1.0.0",
   "scripts": {
-    "build": "sls webpack",
+    "build": "ENVIRONMENT=local sls webpack",
```

Comment:
This is interesting to me - curious if this will come into conflict in the deploy stages.

Reply:
It does feel really weird, but when migrating another repository that used serverless we found it requires the environment variable and it was set to a static value as well.